docs/sql-data-sources-csv.md

```diff
@@ -92,14 +92,14 @@ Data source options of CSV can be set via:
   </tr>
   <tr>
     <td><code>comment</code></td>
-    <td>empty string</td>
+    <td></td>
     <td>Sets a single character used for skipping lines beginning with this character. By default, it is disabled.</td>
     <td>read</td>
   </tr>
   <tr>
     <td><code>header</code></td>
     <td>false</td>
-    <td>For reading, uses the first line as names of columns. For writing, writes the names of columns as the first line. Note that if the given path is a RDD of Strings, this header option will remove all lines same with the header if exists.</td>
```

Comment:
Let's keep this note:

Note that if the given path is a RDD of Strings, this header option will remove all lines same with the header if exists.